tests/browser/features/happy.feature

```diff
@@ -2,10 +2,11 @@ Feature: Happy path
 
   Successful journey through the system and back to the RP
 
-  @mock-api:success
+  @mock-api:success @wip
```

Comment:
Is @wip something we want to commit or was it for testing?

Reply:
Removed!